parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -556,19 +556,24 @@ extractKubeBinaries() {
         echo "cached package ${k8s_tgz_tmp} found, will extract that"
         # remove the current kubelet and kubectl binaries before extracting new binaries from the cached package
         rm -rf /usr/local/bin/kubelet-* /usr/local/bin/kubectl-*
-    else
+    fi
```

Comment:
This does change the logic a bit. If `$is_private_url == true` and `if [[ -n "${k8s_downloads_dir}" ]]`, it will run line 562. 
Before the change, only if `$is_private_url == false` and `if [[ -n "${k8s_downloads_dir}" ]]` will it run the same line.